app/views/products/index.html.haml

```diff
@@ -1,7 +1,10 @@
 %h1 Menu for FoodComrade restaurant
 #ul
   - @products.each do |product|
-    %li
-      = product.name
-      = product.description
-      = product.price
+    %tr
+      %td= product.name
+      %td= product.description
+      %td= product.price
+    - if current_user.try(:admin?)
```

Comment:
why cant you use `current_user.admin?` here?

Reply:
We get undefined method `admin?' for nil:NilClass (ActionView::Template::Error) when we do that. We didn't see why we would need to create a separate Admin model for this particular feature.